src/BigDecimal.php

```diff
@@ -734,19 +738,28 @@ public function toFloat() : float
     }
 
     /**
-     * {@inheritdoc}
+     * @return numeric-string
      */
-    public function __toString() : string
+    public function toNumericString(): string
```

Comment:
Why add this method?

Reply:
Added it to have proper return type (`numeric-string`). Maybe I can remove it and use `@psalm-return` on `__toString()` directly, let me check...